.spelling

```diff
@@ -998,6 +1004,7 @@ Srihari
 Srinath
 Stackdriver
 Standard_D4ds_v5
+StartupProbe
```

Comment:
Nearly all instances of this have backticks as well. I'd remove this and backtick to one remaining instance.